### PR TITLE
Ducktape codestyle

### PR DESCRIPTION
**Pull request by ivandasch**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/ducktests/tests/ignitetest/services/spark.py [hunk 1]
```diff
@@ -103,28 +109,40 @@ def clean_node(self, node):
...
+        """
         if node == self.nodes[0]:
             return "org.apache.spark.deploy.master.Master"
-        else:
-            return "org.apache.spark.deploy.worker.Worker"
 
-    def master_log_path(self, node):
+        return "org.apache.spark.deploy.worker.Worker"
+
+    @staticmethod
```
Comment (timoninmaxim) [inline modules/ducktests/tests/ignitetest/services/spark.py:128]: Do we really need static methods in our code? I don't see any case in our code where usage of them is required. It just add more sugar without need. Also some code styles consider them evil, e.g. [Google Guide](https://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators).

File: modules/ducktests/tests/ignitetest/services/utils/ignite_path.py [hunk 2]
```diff
@@ -13,44 +13,55 @@
...
+        :param script_name: Script name.
+        :param node_or_version: Ignite service node or IgniteVersion instance.
+        :param project: Project name.
+        :return: Full path to script.
+        """
+        version = self.__version__(node_or_version)
         return os.path.join(self.home(version, project=project), "bin", script_name)
 
-    def _version(self, node_or_version):
+    @staticmethod
```
[the remark this answers is not in the document]
Comment (ivandasch) [reply modules/ducktests/tests/ignitetest/services/utils/ignite_path.py:62]: Both pycharm and pylint generate warnings on it.

File: modules/ducktests/tests/ignitetest/services/utils/ignite_path.py [hunk 3]
```diff
@@ -13,44 +13,55 @@
...
+    def __version__(node_or_version):
```
[the remark this answers is not in the document]
Comment (ivandasch) [reply modules/ducktests/tests/ignitetest/services/utils/ignite_path.py:63]: Yep, you are right

[the remark this answers is not in the document]
Comment (ivandasch) [reply modules/ducktests/tests/ignitetest/services/utils/ignite_path.py:63]: Yep, you are right

File: modules/ducktests/tests/ignitetest/services/spark.py (hunk 1 above)
Comment (ivandasch) [reply modules/ducktests/tests/ignitetest/services/spark.py:128]: Despite the fact that in mentioned above codestyle staticmethods are not welcome in cod, linters generate warnings about unused `self`. And usage of `@staticmethod` is a preferrable way to mitigate this case. This is so common in my practice and widely accepted in a python's community.

**Pull request merged**